src/app/components/ItemDetails.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * TCD Software
+ * Created by Dmitrij Rysanow on 03.03.17.
+ */
+
+import React from "react";
+import {Tabs, Tab} from "material-ui/Tabs";
+import SettingsTab from "./SettingsTab";
+import InfoTab from "./InfoTab";
+
+const tabs = [{
+    Id: 0,
+    Title: "info",
+    Component: InfoTab,
+}, {
+    Id: 1,
+    Title: "settings",
+    Component: SettingsTab
+}];
+
+class ItemDetails extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: 0
+        };
+
+    }
+    handleChange = (value) => {
+        console.log("Change state", value);
+        this.setState({
+            value: value
+        });
+    };
+
+    render() {
+        const tabItems = tabs.map((tab, i) => {
```

Comment:
please move this map callback to the constraint or create method in class and put this callback in this method. Result?
Your render method will be look like this
```
render(){
  const tableItems = tabs.map(yourCallbackName);
  return // your code from line 43 
}
```

Isn't it more elegant solution?